app/utils/build-provider-asset-path.js

```diff
@@ -2,7 +2,7 @@ import config from 'ember-get-config';
 import pathJoin from '../utils/path-join';
 
 export default function buildProviderAssetPath(providerId, assetName, isDomain) {
-    if (config.providerAssetsURL) {
+    if (config.providerAssetsURL != 'local') {
```

Comment:
Ohhhhh, I see. providerAssetsURL can either be a URL or can be 'local' as a special case, so it's just the one variable. Yeah, that works, just mention in the ticket description.